boards/arc/emsdp/arc_mpu_regions.c

```diff
@@ -45,6 +45,11 @@ static struct arc_mpu_region mpu_regions[] = {
 			 0xF0000000,
 			 64 * 1024,
 			 REGION_KERNEL_RAM_ATTR),
+	/* Region SPI_1 for SPI_FLASH */
+	MPU_REGION_ENTRY("SPI_1",
```

Comment:
Instead of using another (scarce resource) MPU region, why not enlarge the 0xf000_000 region to include the second SPI instance?

Reply:
Changed to enlarge region to include SPI instance. Example includes emsdp spi loopback and spi-flash sample.